apps/web/public/static/locales/en/common.json

```diff
@@ -720,5 +720,31 @@
   "external_redirect_url": "https://example.com/redirect-to-my-success-page",
   "redirect_url_upgrade_description": "In order to use this feature, you need to upgrade to a Pro account.",
   "duplicate": "Duplicate",
-  "you_can_manage_your_schedules": "You can manage your schedules on the Availability page."
+  "you_can_manage_your_schedules": "You can manage your schedules on the Availability page.",
+  "api_keys": "API Keys",
+  "api_key_modal_subtitle": "API keys allow you to make API calls for your own account.",
+  "api_keys_subtitle": "Generate API keys to use for accessing your own account.",
+  "generate_new_token": "Generate new token",
+  "create_api_key": "Create an API key",
+  "personal_note": "Name this key",
+  "personal_note_placeholder": "E.g. Development",
+  "delete_pat": "Delete personal access token",
+  "api_key_no_note": "Nameless token",
+  "api_key_never_expires":"This API key has no expiration date",
+  "edit_api_key": "Edit token",
+  "never_expire_key": "Never expires",
+  "delete_api_key": "Revoke token",
+  "success_api_key_created": "API key created successfully",
+  "create": "Create",
+  "success_api_key_created_bold_tagline": "Save this token somewhere safe.",
+  "you_will_not_view_once_close_modal": "You will not be able to view it again once you close this modal.",
```

Comment:
NIT:
this can improve to `you_will_only_view_it_once`

Reply:
Hahaha yeah indeed that makes much more sense, I don't know what I was thinking about when I wrote that LOL! fixed on new commit thx alan!